source-web/src/assets/db/photos.json

```diff
@@ -149,5 +149,15 @@
 		"license": "Freemium",
 		"licenseLink": "https://thenounproject.com/pricing/#photos",
 		"licenseDescription": "Free with attribution"
+	},
+	{
+		"id": 17,
+		"order": 8,
+		"name": "Picjumbo",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/f_auto,q_auto/v1/freesets/photos/picjumbo",
+		"link": "https://picjumbo.com/",
+		"license": "Freemium",
+		"licenseLink": "https://picjumbo.com/faq-and-terms/",
+		"licenseDescription": "Free with attribution"
```

Comment:
When using Picjumbo you do not need to give attribution.
![image](https://github.com/cosmoart/Freesets/assets/81999801/51f55d21-c35b-43fa-ac84-937342942dda)

Reply:
I will change that too

Reply:
fixed